client/src/assets/i18n/fr.json

```
@@ -74,11 +74,11 @@
     "ENDAT": "Date de fin",
     "GENERATE-REPORT": "Génération du rapport",
     "DUMPJSON": "Téléchargement des données mission (JSON)",
-    "CODI": "Ajoutez ou modifier le lien codiMD",
+    "CODI": "Ajouter ou modifier un lien codiMD",
```

Actually there can be only one codimd link
```suggestion
    "CODI": "Ajouter ou modifier le lien codiMD",
```

---

done